src/services/gateway.service.ts

```diff
@@ -93,17 +97,25 @@ function createApp(
   )
 }
 
-function updateApp(appId: AppId, updatedAppConfig: AppConfigRequiredProps) {
+function updateApp(appId: AppId, updatedAppConfig: AppState) {
+  const appConfigRequestBody = getAppConfigRequestBody(updatedAppConfig)
   return gatewayAuthorizedInstance.patch(
     `${getEnvApi('v2')}/app/?id=${appId}`,
-    updatedAppConfig
+    appConfigRequestBody
   )
 }
 
-function getAppConfigRequestBody(appId: AppId): AppConfigRequiredProps {
-  let storage_limit: number, bandwidth_limit: number
-  const app = appsStore.app(appId)
+function deleteCred(appId: AppId, authToRemove: SocialAuthState[]) {
+  const deleteCredPromises = authToRemove.map((auth) => {
+    return gatewayAuthorizedInstance.delete(
+      `${getEnvApi('v2')}/cred/?id=${appId}&verifier=${auth.verifier}`
+    )
+  })
+  return Promise.all(deleteCredPromises)
+}
```

Comment:
@shrinathprabhu Do you mind describing the context of this change? Why was it necessary to make separate calls to delete creds instead of making a single update app call? Was this changed on the gateway API?